letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -296,11 +312,24 @@ elif [ -f /etc/mageia-release ]; then
   }
   BOOTSTRAP_VERSION="BootstrapMageiaCommon $BOOTSTRAP_MAGEIA_COMMON_VERSION"
 elif [ -f /etc/redhat-release ]; then
-  Bootstrap() {
-    BootstrapMessage "RedHat-based OSes"
-    BootstrapRpmCommon
-  }
-  BOOTSTRAP_VERSION="BootstrapRpmCommon $BOOTSTRAP_RPM_COMMON_VERSION"
+  prev_le_python="$LE_PYTHON"
```

Comment:
I think I finally figured out what's going on here, but I could use a nice big comment, something like the lines of "We run DeterminePythonVersion to decide, on the basis of available Python versions, whether to use 2.x or 3.x on RedHat-like systems. Then we revert its return values to their previous state."

Reply:
Adding a comment is a good idea. Not reverting PYVER isn't a problem.

Reply:
(I can explain more if you want; it's related to #5388)

Reply:
As long as you have the PYVER reasoning clear in your head, that's good enough for me. :-)